jekyll/_cci2_ja/server/operator/application-lifecycle.adoc

```diff
@@ -1,10 +1,11 @@
 ---
 
-version:
-- Server v4.x
-- サーバー管理者
+contentTags:
+  platform:
+    - Server v4.x
+    - Server Admin
 ---
-= アプリケーションのライフサイクル
+= Application lifecycle
```

Comment:
@yuki-sakamoto-san do you think I should go through this PR and manually revert changes like this that are wrong? A lot of the changes in this PR are good AFAICT